appcenter-data/package.json

```diff
@@ -36,9 +36,22 @@
     "eslint-config-airbnb": "^15.1.0",
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-react": "^7.2.1"
+    "eslint-plugin-react": "^7.2.1",
+    "jest": "^24.8.0",
+    "react": "^16.6.3",
```

Comment:
The new SOP is to remove the `~` and `^` before versions, and you've currently pinned `react-native` when you haven't pinned all the others. My recommendation is to remove all of the prefix and pin everything. @dhei what do you think?

Reply:
@dhei @michaelsmitha I based my PR off of #632. Let’s create another PR to update all of the npm packages.

Reply:
+1 for version pinning for all npm packages.

In the spirit of having smaller PRs, let's do that in a separate PR.